### DISCUSSION TITLE
Missing citations

### DISCUSSION THREAD

**Discussion by prajwel** [astropy/astropy-v5.0-paper#108] [Text comments] [answered]
I went through the Python scripts used to prepare the manuscript. The following used packages were not cited in the manuscript, but they have a "how to cite" page in the documentation. 

Pandas (https://pandas.pydata.org/about/citing.html)
Geopandas (https://geopandas.org/en/stable/about/citing.html?highlight=cite)

**Comment by pllim** [comment]
We do not have to cite "Python scripts used to prepare the manuscript," do we, @adrn ?

**Comment by adrn** [comment, marked as answer]
~~Yes I think we should cite these packages in the `\software{}` section. @prajwel would you be willing to open a PR to add these?~~
I added the references and pushed.
